utils/model.py

```
@@ -0,0 +1,108 @@
+                **inputs,
+                max_new_tokens=50,
+                do_sample=False,
+                temperature=0.1,  # 일관성을 위해 낮게 설정
+                pad_token_id=tokenizer.eos_token_id,
+                eos_token_id=tokenizer.eos_token_id
+            )
+        
+        full_text = tokenizer.decode(outputs[0], skip_special_tokens=True)
+        return full_text[len(prompt):].strip() if full_text.startswith(prompt) else full_text
+            
+    except Exception as e:
```

Ruff (0.11.9)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)